app/plugins/eu.esdihumboldt.hale.app.cli.commands/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: hale CLI Default Commands
 Bundle-SymbolicName: eu.esdihumboldt.hale.app.cli.commands;singleton:=true
-Bundle-Version: 5.0.0.qualifier
+Bundle-Version: 5.1.0.qualifier
```

Comment:
I guess the versions should match the release versions? i.e. 5.0.1 instead of 5.1.0?